### ISSUE TITLE
Fast switching between files makes the extension thinks I'm using it in an inappropriate way

### ISSUE THREAD

**Issue by timsu92** [microsoft/pylance-release#5052]
Type: <b>Bug</b>

# Behaviour
## Expected vs. Actual

I was switching my files fast, and the extension failed to start, making it thinks I'm not using in a copy of Visual Studio Code.

## Steps to reproduce:

1. Open VSCode. For example, I've had a `ipynb` file opened
2. Click on another `ipynb` file.
3. While plugin initialization isn't completed, switch to another `ipynb` file
4. Notification shows up, saying I'm using a copy of VSCode, and this plugin isn't licensed for that, while I'm truly using VSCode.
5. Further modifications to my code isn't assisted by this plugin anymore

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.10.12
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Global
-   Value of the `python.languageServer` setting: Default

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
2023-10-29 18:44:08.225 [info] Test server listening.
2023-10-29 18:44:08.225 [info] > conda info --json
2023-10-29 18:44:08.225 [info] > poetry env list --full-path
2023-10-29 18:44:08.225 [info] cwd: .
2023-10-29 18:44:08.225 [info] Python 解譯器路徑: ./.venv/bin/python
2023-10-29 18:44:08.295 [info] Found: /bin/python3 --> /bin/python3.10
2023-10-29 18:44:08.304 [info] Found: /bin/python3.10 --> /bin/python3.10
2023-10-29 18:44:08.598 [info] Found: /usr/bin/python3 --> /usr/bin/python3.10
2023-10-29 18:44:08.608 [info] Found: /usr/bin/python3.10 --> /usr/bin/python3.10
2023-10-29 18:44:08.631 [info] > ./.venv/bin/python -I ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/get_output_via_markers.py ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/interpreterInfo.py
2023-10-29 18:44:08.645 [info] > . ./.venv/bin/activate && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/printEnvVariables.py
2023-10-29 18:44:08.645 [info] shell: bash
2023-10-29 18:44:08.652 [info] > . ./.venv/bin/activate && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/printEnvVariables.py
2023-10-29 18:44:08.652 [info] shell: bash
2023-10-29 18:44:08.660 [info] > . ./.venv/bin/activate && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/printEnvVariables.py
2023-10-29 18:44:08.660 [info] shell: bash
2023-10-29 18:44:08.972 [info] > /bin/python3 -I ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/get_output_via_markers.py ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/interpreterInfo.py
2023-10-29 18:44:09.002 [info] > /usr/bin/python3 -I ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/get_output_via_markers.py ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/interpreterInfo.py
2023-10-29 18:44:09.436 [info] 正在啟動 Pylance 語言伺服器。
2023-10-29 18:44:42.488 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:48:13.537 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:48:20.978 [info] Python 解譯器路徑: /bin/python3
2023-10-29 18:49:00.275 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:50:05.927 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:51:49.087 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:52:21.181 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:52:40.435 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:52:41.997 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:52:59.905 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:53:12.905 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:58:52.067 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:58:57.998 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 18:59:49.855 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:00:21.198 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:00:25.177 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:00:26.355 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:00:27.688 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:00:32.218 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:00:33.236 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:00:34.061 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:01:12.307 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:01:30.857 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:02:14.165 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:02:21.826 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:02:23.957 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:02:24.698 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:02:58.488 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:03:01.267 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:03:01.807 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:03:02.275 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:03:06.986 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:05:17.264 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:05:18.865 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:05:19.819 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:05:20.375 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:06:30.416 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:06:30.911 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:06:41.465 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:07:00.677 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:07:30.078 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:07:32.976 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:45:11.505 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:45:20.658 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:45:28.233 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:45:28.978 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:45:34.708 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:45:51.109 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:45:57.729 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:45:58.428 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:46:19.453 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:47:53.774 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:48:00.299 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:48:01.157 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:48:01.670 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:50:03.385 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:50:17.080 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:51:03.599 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:51:04.289 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:53:44.958 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:53:58.092 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:55:27.371 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:55:42.630 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:57:38.980 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:57:41.266 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:57:43.267 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:57:46.636 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:58:10.804 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:58:21.390 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:58:37.146 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:58:37.795 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:58:38.689 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:58:40.191 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 19:59:09.759 [info] > . ./.venv/bin/activate && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/printEnvVariables.py
2023-10-29 19:59:09.759 [info] shell: bash
2023-10-29 19:59:09.789 [info] > . ./.venv/bin/activate && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python ~/.vscode-server/extensions/ms-python.python-2023.18.0/pythonFiles/printEnvVariables.py
2023-10-29 19:59:09.789 [info] shell: bash
2023-10-29 20:01:29.449 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 20:03:01.249 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 20:03:06.635 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 20:03:07.230 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 20:03:34.051 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
2023-10-29 20:04:57.481 [info] Discover tests for workspace name: undefined - uri: c:/Users/timsu/AppData/Roaming/Code/User/settings.json
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```

languageServer: "Pylance"

```

</p>
</details>

Extension version: 2023.18.0
VS Code version: Code 1.83.1 (f1b07bd25dfad64b0167beb15359ae573aecd2cc, 2023-10-10T23:48:05.904Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Remote OS version: Linux x64 5.15.90.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-12700H (14 x 2688)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.63GB (15.48GB free)|
|Process Argv|--folder-uri vscode-remote://wsl%2Bubuntu-22.04/home/tim/1121aicp --crash-reporter-id 76077254-8c7e-4f22-9ca8-d18f2ed5db71|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-22.04|
|OS|Linux x64 5.15.90.1-microsoft-standard-WSL2|
|CPUs|12th Gen Intel(R) Core(TM) i7-12700H (14 x 2688)|
|Memory (System)|15.44GB (12.87GB free)|
|VM|0%|
</details>

**Comment by timsu92** [comment]
The above information saying my Python virtual environment is `global` is wrong. Actually, I'm using one managed my [Poetry](https://python-poetry.org/).

The above template asks for `Python` output. However, those information isn't helpful. I guess the output of Python language server helps, so I'm pasting here. But I have no idea why this message seems to be cropped that I'm not able to copy any messages older😢

<details>

<summary>Output for <code>Python language server</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python language server</code>)
</summary>

```
Error: Debug Failure. Unable to find notebook URI for vscode-notebook-cell://wsl%2Bubuntu-22.04/home/tim/1121aicp/%E6%9C%9F%E4%B8%AD%E8%80%83/Q14%20CSV%20file.ipynb#W1sdnNjb2RlLXJlbW90ZQ%3D%3D
    at _0x4f6f08._getNotebookUri (/home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:377799)
    at _0x4f6f08.getNotebookPathFromCell (/home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:375944)
    at _0x4f6f08.getNotebookCellPath (/home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:375163)
    at _0x29682b.decodeTextDocumentUri (/home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:347323)
    at _0x5e13d0.onDidCloseTextDocument (/home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:1035718)
    at /home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:1143599
    at /home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/vendor.bundle.js:2:1471770
    at oe (/home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/vendor.bundle.js:2:1471992)
    at /home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/vendor.bundle.js:2:1467932
    at Immediate.<anonymous> (/home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/vendor.bundle.js:2:1467952)
    at processImmediate (node:internal/timers:476:21)

2023-10-29 20:06:18.684 [info] [Info  - 下午8:06:18] Connection to server got closed. Server will restart.
2023-10-29 20:06:18.684 [info] true
2023-10-29 20:06:19.067 [info] [Info  - 下午8:06:19] (22382) Pylance language server 2023.10.50 (pyright cfe56bfb) starting
2023-10-29 20:06:19.067 [info] [Info  - 下午8:06:19] (22382) Server root directory: /home/tim/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist
2023-10-29 20:06:19.073 [info] [Error - 下午8:06:19] Server initialization failed.
2023-10-29 20:06:19.073 [info]   Message: Request initialize failed with message: You may install and use any number of copies of the software only with Microsoft Visual Studio, Visual Studio for Mac, Visual Studio Code, Azure DevOps, Team Foundation Server, and successor Microsoft products and services (collectively, the “Visual Studio Products and Services”) to develop and test your applications. The software is licensed, not sold. This agreement only gives you some rights to use the software. Microsoft reserves all other rights. You may not: work around any technical limitations in the software that only allow you to use it in certain ways; reverse engineer, decompile or disassemble the software, or otherwise attempt to derive the source code for the software, except and to the extent required by third party licensing terms governing use of certain open source components that may be included in the software; remove, minimize, block, or modify any notices of Microsoft or its suppliers in the software; use the software in any way that is against the law or to create or propagate malware; or share, publish, distribute, or lease the software (except for any distributable code, subject to the terms above), provide the software as a stand-alone offering for others to use, or transfer the software or this agreement to any third party.
  Code: -32603 
2023-10-29 20:06:19.073 [info] [Error - 下午8:06:19] Pylance client: couldn't create connection to server.
2023-10-29 20:06:19.073 [info]   Message: Request initialize failed with message: You may install and use any number of copies of the software only with Microsoft Visual Studio, Visual Studio for Mac, Visual Studio Code, Azure DevOps, Team Foundation Server, and successor Microsoft products and services (collectively, the “Visual Studio Products and Services”) to develop and test your applications. The software is licensed, not sold. This agreement only gives you some rights to use the software. Microsoft reserves all other rights. You may not: work around any technical limitations in the software that only allow you to use it in certain ways; reverse engineer, decompile or disassemble the software, or otherwise attempt to derive the source code for the software, except and to the extent required by third party licensing terms governing use of certain open source components that may be included in the software; remove, minimize, block, or modify any notices of Microsoft or its suppliers in the software; use the software in any way that is against the law or to create or propagate malware; or share, publish, distribute, or lease the software (except for any distributable code, subject to the terms above), provide the software as a stand-alone offering for others to use, or transfer the software or this agreement to any third party.
  Code: -32603 
2023-10-29 20:06:19.073 [info] [Error - 下午8:06:19] Restarting server failed
2023-10-29 20:06:19.073 [info]   Message: Request initialize failed with message: You may install and use any number of copies of the software only with Microsoft Visual Studio, Visual Studio for Mac, Visual Studio Code, Azure DevOps, Team Foundation Server, and successor Microsoft products and services (collectively, the “Visual Studio Products and Services”) to develop and test your applications. The software is licensed, not sold. This agreement only gives you some rights to use the software. Microsoft reserves all other rights. You may not: work around any technical limitations in the software that only allow you to use it in certain ways; reverse engineer, decompile or disassemble the software, or otherwise attempt to derive the source code for the software, except and to the extent required by third party licensing terms governing use of certain open source components that may be included in the software; remove, minimize, block, or modify any notices of Microsoft or its suppliers in the software; use the software in any way that is against the law or to create or propagate malware; or share, publish, distribute, or lease the software (except for any distributable code, subject to the terms above), provide the software as a stand-alone offering for others to use, or transfer the software or this agreement to any third party.
  Code: -32603
```

</details>

**Comment by rchiodo** [comment]
Thanks for the bug. This should be a dupe of #5009 which we fixed in 2023.10.53.

**Comment by timsu92** [comment]
Thanks for transferring this issue and duplication searching! I didn't know that I should search for this in Pylance.
